docs/core/extensions/snippets/configuration/worker-service-options/WorkerWithOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace WorkerServiceOptions.Example
+{
+    public class WorkerWithOptions : BackgroundService
+    {
+        bool _hasWrittenSettings;
+        readonly ILogger<WorkerWithOptions> _logger;
+        readonly CustomSettings _settings;
```

Comment:
I'd prefer these to be explicitly `private`:

```suggestion
        private bool _hasWrittenSettings;
        private readonly ILogger<WorkerWithOptions> _logger;
        private readonly CustomSettings _settings;
```